docs/getting-started-guides/windows/README.md

```diff
@@ -0,0 +1,159 @@
+
+#### Linux
+
+Linux hosts should be setup according to their respective distro documentation and the requirements of the Kubernetes version you will be using. Linux hosts also require to have CNI Setup.
+
+### Component Setup
+In order to build the work node components, the *kubelet* and *kube-proxy* for Windows, the following needs to be installed on the host
+* Git, Go 1.7.1+ 
+* make (if using Linux or MacOS)
+* Important notes and other dependencies are listed [here](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md#building-kubernetes-on-a-local-osshell-environment)
+
+#### kubelet
+
+In order to build the *kubelet*, run:
+
+1. `cd $GOPATH/src/k8s.io/kubernetes`
+2. Build *kubelet*
+   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet`
+   2. Windows: `go build cmd/kubelet/kubelet.go`
+
+#### kube-proxy
+
+In order to build *kube-proxy*, run:
+
+1. `cd $GOPATH/src/k8s.io/kubernetes`
+2. Build *kube-proxy*
+   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kube-proxy`
+   2. Windows: `go build cmd/kube-proxy/proxy.go`
+
+### Route Setup
+
+The below example setup assumes one Linux and two Windows Server 2016 nodes and a cluster CIDR 192.168.0.0/16
+
+| Hostname | Routable IP address | Pod CIDR |
+| --- | --- | --- |
+| Lin01 | `<IP of Lin01 host>` | 192.168.0.0/24 |
+| Win01 | `<IP of Win01 host>` | 192.168.1.0/24 |
+| Win02 | `<IP of Win02 host>` | 192.168.2.0/24 |
+
+**Lin01**
+```
+ip route add 192.168.1.0/24 via <IP of Win01 host>
+ip route add 192.168.2.0/24 via <IP of Win02 host>
+```
+
+**Win01**
+```
+docker network create -d transparent --gateway 192.168.1.1 --subnet 192.168.1.0/24 <network name>
+# A bridge is created with Adapter name "vEthernet (HNSTransparent)". Set its IP address to transparent network gateway
+netsh interface ipv4 set address "vEthernet (HNSTransparent)" addr=192.168.1.1
+route add 192.168.0.0 mask 255.255.255.0 192.168.0.1 if <Interface Id of the Routable Ethernet Adapter> -p
+route add 192.168.2.0 mask 255.255.255.0 192.168.2.1 if <Interface Id of the Routable Ethernet Adapter> -p
+```
+
+**Win02**
+```
+docker network create -d transparent --gateway 192.168.2.1 --subnet 192.168.2.0/24 <network name>
+# A bridge is created with Adapter name "vEthernet (HNSTransparent)". Set its IP address to transparent network gateway
+netsh interface ipv4 set address "vEthernet (HNSTransparent)" addr=192.168.2.1
+route add 192.168.0.0 mask 255.255.255.0 192.168.0.1 if <Interface Id of the Routable Ethernet Adapter> -p
+route add 192.168.1.0 mask 255.255.255.0 192.168.1.1 if <Interface Id of the Routable Ethernet Adapter> -p
+```
+
+## Starting the Cluster
+For now, the Kubernetes control plane continues to run on Linux and as a result a Windows only Kubernetes Cluster is not possible. 
+## Linux
+Use your preferred method to start Kubernetes cluster on Linux. Please note that Cluster CIDR might need to be updated.
+## Windows
+### kubelet
```

Comment:
Nested headings. Replace "### kubelet" with "To start kubelet on your Windows node:"